client/src/components/History/CurrentHistory/HistoryOperations/SelectionOperations.vue

```diff
@@ -402,5 +402,6 @@ export default {
 <style>
 .modal-with-selector {
     overflow: initial;
+    height: 300px; /* To make room for the selector */
```

Comment:
This doesn't ever grow larger than 300, does it?  (i.e., should this be `min-height`?)

Reply:
Sorry for the late reply. I tried with min-height, but it doesn't change anything since these 2 modals only have the selector and nothing else and they don't grow larger even if you open the dropdown. I will use min-height anyway :+1:

Reply:
No worries, I wasn't asking for any changes really—just curious about that idea. Thanks for the quick fix and followup; it looks great!